### PR TITLE
Welcome page 

### PR DESCRIPTION
**Pull request by GALTdea**
## What's the change?
Added Homepage (Hero, Testimonial,Problem) sections.

## Highlights / Surprises / Risks / Cleanup
Needed to add a couple of helpers to conditionally render navbars and style for the welcome page.

## Demo / Screenshots
Screen Shot 2023-08-14 at 5 12 59 PM
Screen Shot 2023-08-14 at 5 13 15 PM
Screen Shot 2023-08-14 at 5 13 23 PM

Mobile:
Screen Shot 2023-08-14 at 5 16 02 PM
Screen Shot 2023-08-14 at 5 16 21 PM
Screen Shot 2023-08-14 at 5 16 32 PM
8-4a2b-b466-c6f2e8562a0d)

### CODE REVIEW

File: app/views/landing/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+              <strong>Real code, real peers</strong>
+              <p class="text-[rgba(255,255,255,0.678)]">Work together on software projects on a team with other developers. Get feedback on your code from experienced engineers. Talk to users.</p>
+            </div>
+
+          </div>
+        </div>
+        <div class="flex justify-center mb-5 leading-loose text-base font-medium">
+          <div class="flex justify-center gap-1 flex-col md:flex-row">
+            <div class="p-2 w-full md:w-1/2 max-w-prose mx-auto flex-1">
+              <strong>Mastermind Community</strong>
```
Comment (JoshDevHub) [inline app/views/landing/index.html.erb:176]: So I realize this is how it is on the current main site, but it feels weird to me how on mobile, you get 2 paragraphs together, then two images.
Selection_243
Selection_244

You can fix this but still keep the staggered layout on desktop by changing `flex-col` to `flex-col-reverse` on line 174. And you'd want to do this same thing further down the page where this also occurs.

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Looks pretty good. I was also maybe wondering about the image sizing in the testimonials section, as that was discussed in the meeting today.

Not necessarily a blocker, but I also think it'd help some if this file broke some of its reusable sections out into partials or something. Would make the markup easier to reason about and would help maintain consistency. That can be done later though if there's no bandwidth to deal with it now.

File: app/views/landing/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+      </div>
+      <div class="flex w-full ">
+        <p class="mb-5 text-[rgba(255,255,255,0.678)]">Join our private community of highly motivated early career developers.</p>
+      </div>
+      <!-- CTA's -->
+      <div class="flex-col w-full my-5 ">
+        <div class="sm:w-3/4  mb-4">
+          <button class="flex justify-between items-center w-full h-[70px] mb-4 bg-[#38D68E] text-[#363942] font-bold px-4 rounded-lg hover:scale-105 transform transition-transform">
+              Become a Member
+              <svg xmlns="http://www.w3.org/2000/svg" height="1.5em" viewBox="0 0 448 512">
```
[the remark this answers is not in the document]
Comment (toyhammered) [reply app/views/landing/index.html.erb:20]: Second to this, please make sure to use font awesome icons.

[the remark this answers is not in the document]
Comment (GALTdea) [reply app/views/landing/index.html.erb:20]: I just changed all the svg's to font awesome classes but haven't had a chance to figure out how abstract out into a partial. I'll try to work on it today, but not sure I can get to it on time. If ready we can merge this and I can  submit a new PR for that

[the remark this answers is not in the document]
Comment (JoshDevHub) [reply app/views/landing/index.html.erb]: I just wouldn't worry about it. We can bring it up in a new ticket later. Shipit :boat:

**Review by JoshDevHub — APPROVED** [review]
LGTM

**Pull request merged**